.github/workflows/pkgci_sharktank_regression.yml

```diff
@@ -0,0 +1,97 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI Sharktank Regression Tests
```

Comment:
We can add more information to https://iree.dev/developers/general/testing-guide/#sharktank-tests about these tests now. Should mention the benchmarking at least.